<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="rBot2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>等额本金</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBot2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="l7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>贷款利率：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="box2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>按面积和单价计算</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按贷款总额计算</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="l_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(元/平米）</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>商业贷款/公积金贷款</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>组合型贷款</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="l5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按揭成数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>（平方米）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>按揭年数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计算方式：</string>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>贷款类别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="l8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>还款方式：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBot1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始计算</string>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>房屋单价：</string>
    </property>
   </widget>
   <widget class="QLabel" name="l4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>房屋面积：</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>（成）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBot1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>等额本息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBot3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(年）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pBot2</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
